README.md

```diff
@@ -147,6 +147,17 @@ See [conform.format()](#formatopts-callback) for more details about the paramete
 
 To view configured and available formatters, as well as to see the path to the log file, run `:ConformInfo`
 
+To see all preset configurations, you can see [here](https://github.com/stevearc/conform.nvim/tree/master/lua/conform/formatters), you just need add filename to your config for `conform`, like this:
+
+```lua
+-- for prettier_d
+require("conform").setup({
+    formatters_by_ft = {
+        javascript = { "prettierd"},
+    },
+})
+```
+
```

Comment:
The list of formatters is literally right below this, and the code example seems duplicative of the other examples. I don't think this note is necessary.

Reply:
ok, I will remove it